common/interpreter/gipsy_scheme/packages/package.scm

```diff
@@ -0,0 +1,160 @@
+(define (caar x) (car (car x)))
+(define (cadr x) (car (cdr x)))
```

Comment:
same... this is built automatically & should not be included in the repository.

Reply:
thank you. as we discussed, this is part of the tiny scheme fix.